container/src/main/java/com/flipkart/poseidon/api/Configuration.java

```diff
@@ -42,4 +42,6 @@
     public JettyConfiguration getJettyConfiguration();
 
     public ExceptionMapper getExceptionMapper();
+
+    public Headers getConfiguredHeaders();
```

Comment:
getConfiguredHeaders() doesn't sound right. Do you want to change it to getProxyHeadersConfiguration() or something like that?